107/src/main/java/org/ehcache/jsr107/Eh107CacheManager.java

```diff
@@ -219,6 +224,22 @@ public Properties getProperties() {
       cacheLoaderWriterFactory.registerJsr107Loader(cacheName, cacheLoaderWriter);
     }
 
+    //If cache is not read or write thro' remove the loader writer configuration from the JSR 107 conf
+    if(!jsr107Config.isReadThrough() && !jsr107Config.isWriteThrough()) {
+      DefaultCacheLoaderWriterConfiguration conf = builder.getExistingServiceConfiguration(DefaultCacheLoaderWriterConfiguration.class);
+      if(conf != null) {
+        LOG.warn("Removing the loader/writer service configuration from the JSR107 cache {}", cacheName);
```

Comment:
Do not comment on what you are doing, but why. Something like: "Ignoring template provided CacheLoaderWriter configuration"